app/services/SessionService.scala

```diff
@@ -36,5 +36,14 @@ class SessionService @Inject()() {
       Right(result.addingToSession(key -> value))
     }
   }
+
+  def remove(keys: Seq[String], result: Result)(implicit user: MtdItUser[_], ec: ExecutionContext): Future[Either[Throwable, Result]] = {
```

Comment:
should be fine to set all required values in the SessionService to empty; I don't think we actually would have to delete anything; record would expired and will be no longer available

Reply:
I think this is maybe an issue to be determined when the mongo implementation is done? The person who does that can decide how they want removal to work